tests/ui/base_test.py

```diff
@@ -30,3 +54,184 @@ def setContext(self):
   def tearDown(self):
     """Teardown for test methods."""
     self.driver.quit()
+
+  def add_test_user_helper(self):
+    """Manually add a test user once the tabs are displayed."""
+    add_manually_form = WebDriverWait(self.driver, 10).until(
+        EC.visibility_of_element_located(((UfOPageLayout.ADD_MANUALLY_FORM))))
+    name_paper_input = add_manually_form.find_element(
+        *UfOPageLayout.ADD_MANUALLY_INPUT_NAME)
+    name_input = name_paper_input.find_element(By.ID, 'input')
+    name_input.send_keys(BaseTest.TEST_USER_AS_DICT['name'])
+    email_paper_input = add_manually_form.find_element(
+        *UfOPageLayout.ADD_MANUALLY_INPUT_EMAIL)
+    email_input = email_paper_input.find_element(By.ID, 'input')
+    email_input.send_keys(BaseTest.TEST_USER_AS_DICT['email'])
+    submit_button = self.driver.find_element(
+        *UfOPageLayout.ADD_MANUALLY_SUBMIT_BUTTON)
+    submit_button.click()
+
+    # Wait for post to finish, can take a while.
+    WebDriverWait(self.driver, 20).until(
+        EC.invisibility_of_element_located(((
+            UfOPageLayout.ADD_MANUALLY_SPINNER))))
+
+  def remove_test_user(self, raiseException=True):
+    """Manually remove a test user using the landing page (the only way).
+
+    Args:
+      raiseException: True to raise an exception if the user is not found.
+    """
+    # Find the user and navigate to their details page.
+    self.driver.get(self.args.server_url + flask.url_for('landing'))
+    landing_page = LandingPage(self.driver)
+    user_list_item = landing_page.GetElement(LandingPage.USER_LIST_ITEM)
+    user_listbox = user_list_item.find_element(*LandingPage.GENERIC_LISTBOX)
+    user_item = self.find_item_in_listing(
+        user_listbox, BaseTest.TEST_USER_AS_DICT['name'])
+
+    if user_item is None:
+      if raiseException:
+        raise Exception
+      else:
+        return
+    else:
+      user_item.click()
+
+    # Click delete on that user.
+    details_modal = user_item.find_element(*LandingPage.DETAILS_MODAL)
+    WebDriverWait(self.driver, 10).until(
+        EC.visibility_of(details_modal))
+    delete_button = details_modal.find_element(*LandingPage.USER_DELETE_BUTTON)
+    delete_button.click()
+
+    # Wait for post to finish, can take a while.
+    WebDriverWait(self.driver, 20).until(
```

Comment:
QQ: why is it that we setting a timeout of 20 here, but 10 in line 103?  Why not pick something consistent, say 30 seconds across the board?  I'm just worried that something short like 10 might cause flakiness in case the server is loaded or otherwise slow.  30 usually is what people have settled on, since it's long enough, yet the test will move on when the EC is fulfilled.